### ISSUE TITLE
Enhance docs with mention of prebundled Python and libs

### ISSUE THREAD

**Issue by Wkalmar** [HelloZeroNet/ZeroNet#2245]
**Is your feature request related to a problem? Please describe.**
In overall I've enjoyed ZeroNet very much. However, when I've reached "How can I create a ZeroNet site" section I've found non-obvious that instead of using `zeronet.py` as a Windows user I could use `zeronet.cmd` from `\lib` folder. Since I've found pip package manager UX quite frustrating I think that this is a thing that should be mentioned as early as possible and could save a couple of hours for other windows users.

**Describe the solution you'd like**
I guess "Windows" subsection in "How can I create a ZeroNet site?" would be sufficient. 

**Describe alternatives you've considered**
IDK. You could tell me where you would like to see it.

I hope you're having a good day :)
Regards,
Bohdan

**Comment by purplesyringa** [comment]
By the way, it's *not* recommended to use command line to create sites. Please use "Create new, empty site" button on ZeroHello instead.

**Comment by Wkalmar** [comment]
I guess that just proves my point: no one can infere this from readme.md

**Comment by ghost** [comment]
Yeah, the readme is outdated, but the docs (at zeronet.io/docs) is much more up to date

**Comment by HelloZeroNet** [comment]
I just added a modification that makes the command-line siteCreate to create site from user's master seed in Rev4253 (but it's still easier to use the web interface to do it)

**Comment by jamesforbes88** [comment]
Hello Everyone 
Have you heard about forex binary tradings well it’s the latest in the market we can help you trade you funds while you sit back at home and watch your chart grow successfully we have various platform you can visit our web at http://fxgrandtrade.com/
Thank you.

**Comment by purplesyringa** [comment]
GitHub issue ads is something new...
